sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/DiscountServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package com.exadel.sandbox.team5.service.impl;
+
+import com.exadel.sandbox.team5.dao.DiscountDAO;
+import com.exadel.sandbox.team5.entity.Discount;
+import com.exadel.sandbox.team5.service.DiscountService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Transactional
+@Service
+@RequiredArgsConstructor
+public class DiscountServiceImpl implements DiscountService {
```

Comment:
It will be great if we create abstract `CRUDServiceImpl` class as well.

Reply:
Have another task for this